Add adopt pack page (#395)
demisto/content-docs: docs/partners/adopt.md

---
id: adopt 
title: Adopt-a-Pack 
---

> The Cortex XSOAR Adopt-a-Pack program provides our technical partners a way ‘take over’ a pack that was originally written by Demisto or Palo Alto Networks. The partner becomes the maintainer and supporter of the pack and receives sevral benefits as outlined below.

Reviewer: ```suggestion
The Cortex XSOAR Adopt-a-Pack program provides our technical partners a way to ‘take over’ a pack that was originally written by Cortex XSOAR developers. The partner becomes the maintainer and supporter of the pack and receives several benefits as outlined below.
```

Author: I want to leave Demisto in there since the pack may be old.... I'll make other changes.

## Benefits for our Partner
Adopting a Pack has several advantages for the partner:
- The partner directly controls the pack’s roadmap, features and release timing.
- The partner receives direct feedback from customers in the form of defects and enhancement requests.
- The partner is added as a reviewer in GitHub so they can see all community updates to that pack.
- The partner places their name on the pack.
- The partner can engage in marketing activities with Palo Alto Networks.

## Process
The process to Adopt-a-Pack is simple:
- Partner signs the Palo Alto Networks Technology Partnership Agreement. If you've already signed a similar agreement, you will not have to sign again. 
- Partner notifies Palo Alto Networks that they wish to adopt the pack. *Important:* please notify us so we can work with you. 
- Partner clones existing pack and updates the pack (improves marketing, adds new commands/APIs, adds playbooks, adds support information, etc.).
- Partner submits a Pull Request adding their new pack to the Cortex XSOAR Marketplace. Usually we add "v2" to the end of the name. 

> - Palo Alto Networks will mark the original pack as Deprecated and direct customers to the new pack. Palo Alto Networks will still maintain the old pack and support existing customers that do not wish to migrate to the new pack. Maintenance will only include bug fixes and will not include new features or enhancements.

Reviewer: This is what we have agreed? I am not sure this is what we want. I think it is better if they will take the original pack unless they insist not to, in that case, we will deprecate the old one and add v2 to the new one.

Author: If no one is using the pack, we can allow them to edit the master and take it over. But if a customer is using it (and we may not even know), then we need to allow the customer to choose to keep using the existing pack with PAN support. Had a call with a customer who expressed issue with us changing support under him. Guy, Anar and Jonathan are onboard.

Reviewer: I can understand their concern... ok.

The process to Adopt-a-Pack is simple:
- Partner signs the Palo Alto Networks Technology Partnership Agreement. If you've already signed a similar agreement, you will not have to sign again. 
- Partner notifies Palo Alto Networks that they wish to adopt the pack. *Important:* please notify us so we can work with you. 

> - If customers are using the existing pack, Partner clones existing pack and updates the cloned pack (improves marketing, adds new commands/APIs, adds playbooks, adds support information, etc.). If the existing pack is not in use, the Partner can edit the existing pack directly. 

Reviewer: Having said all of the previous discussion do you want to keep the discussion about the pack usage open? How do partners even know this info?
